libraries/rush-lib/src/logic/operations/PhasedOperationPlugin.ts

```diff
@@ -40,46 +41,63 @@ function createOperations(
     }
   }
 
+  for (const operation of operationsWithChanges) {
+    for (const consumer of operation.consumers) {
+      operationsWithChanges.add(consumer);
+    }
+  }
+
+  for (const [key, operation] of operations) {
+    if (!operationsWithChanges.has(operation)) {
+      // In scope, but not changed.
+      operation.runner = new NullOperationRunner({
+        name: key,
+        result: OperationStatus.FromCache,
```

Comment:
Seems like this should be `Skipped`, since it didn't get restored from cache.

Reply:
It did get restored from cache. It was restored from cache by virtue of that it was built by the currently executing process and has not changed since. Perhaps we need to add yet another status. Long term when I revamp I'll be preserving the prior state fully; this PR is a stopgap to allow `isolatedModules` and resulting dependency-less phases to do the correct amount of work on a rebuild.

Reply:
Let's use NoOp here.

Reply:
I just realized that it is not required to have succeeded in the last build attempt, so I'll continue with `Skipped` for now.